src/etc/inc/ipsec.inc

```diff
@@ -2132,6 +2136,17 @@ function ipsec_setup_tunnels() {
 			$conn['tfc_padding'] = (isset($ph1ent['tfc_bytes']) && is_numericint($ph1ent['tfc_bytes'])) ? $ph1ent['tfc_bytes'] : "mtu";
 		}
 
+		/* Custom Port */
+		if (isset($ph1ent['ikeport'])) {
+			if (isset($config['ipsec']['port_nat_t'])) {
+				$conn['leftikeport'] = $config['ipsec']['port_nat_t'];
+			}
+			else {
+				$conn['leftikeport'] = 4500;
+			}
```

Comment:
Simplify similar to above comments:
```
$conn['leftikeport'] = isset($config['ipsec']['port_nat_t']) ? $config['ipsec']['port_nat_t'] : 4500;
```

Reply:
On master pfSense is using swanctl.conf format, not ipsec.conf, so the directive is `local_port`: https://wiki.strongswan.org/projects/strongswan/wiki/Fromipsecconf